axoupdater/src/release/github.rs

```diff
@@ -149,27 +149,39 @@ pub(crate) async fn get_github_releases(
     let mut data: Vec<Release> = vec![];
 
     while pages_remain {
+        // fetch the releases
         let resp = get_releases(&client, &url, token).await?;
 
+        // collect the response headers
         let headers = resp.headers();
-        let link_header = &headers[reqwest::header::LINK]
-            .to_str()
-            .expect("header was not ascii")
-            .to_string();
-        pages_remain = link_header.contains("rel=\"next\"");
+        let link_header_val = &headers.get(reqwest::header::LINK);
+        let link_header = link_header_val.as_ref().map(|link_header_val| {
```

Comment:
Minor nit: we probably don't actually have to assign `link_header_val` above and could just chain this here (which'd make the reuse of the `link_header_val` variable name a little clearer)

Reply:
sure thing